yjit/src/virtualmem.rs

```diff
@@ -198,6 +198,7 @@ impl CodePtr {
         ptr as i64
     }
 
+    #[cfg(target_arch = "aarch64")]
```

Comment:
This is a convenience function so I don't think we should guard it like that. Otherwise someone might try to use it, wonder why it's not working, maybe even reimplement it.

Reply:
Yeah, and if they do, they will see it exists and just remove the flag here. The alternative would be to just say that it is allowed to be unused. I'm fine with that.